arclines/data/NIST/XeI_vacuum.ascii

```diff
@@ -1,6 +1,6 @@
-# Retreived from NIST by JXP on 14 Apr 2016
-# Vacuum, 4000-12000Ang
-# Min Rel Int = 1000.
+# Retreived from NIST by JXP on 03 May 2018
+# Vacuum, 8300-12000Ang
```

Comment:
But the list goes down to 8200A - Also, I'm guessing there are no lines blueward of this, which is why we are not taking from 4000 anymore?

Reply:
Edited to 8200.   And, yes, I don't know of anyone
vetting XeI down below 8000A.